app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalArticleController.php

```diff
@@ -60,6 +61,14 @@ public function show(KnowledgeBaseCategory $category, KnowledgeBaseItem $article
                 'name' => $article->title,
                 'lastUpdated' => $article->updated_at->format('M d Y, h:m a'),
                 'content' => tiptap_converter()->asHTML(TiptapMediaEncoder::decode($article->article_details)),
+                'tags' => Feature::active('tags') ? $article->tags()
+                    ->orderBy('name')
+                    ->get()
+                    ->select([
+                        'id',
+                        'name',
+                    ])
```

Comment:
I think if you put the `select` before the `get` here it will make sure to actually scope the query to the DB, reducing the data in SQL.